src/gen/gen_eval_files.lua

```diff
@@ -797,6 +798,23 @@ local function get_option_meta()
       end
       local r = vim.deepcopy(o) --[[@as vim.option_meta]]
       r.desc = o.desc:gsub('^        ', ''):gsub('\n        ', '\n')
+      if o.full_name == 'eventignorewin' then
+        local events = require('nvim.auevents').events
+        local tags_file = assert(io.open(TAGS_FILE))
+        local tags_text = tags_file:read('*a')
+        tags_file:close()
+        local map_fn = function(k, v)
+          if v then
+            return nil
+          end
+          local tag_pat = ('\n%s\t([^\t]+)\t'):format(k)
+          local link_text = ('|%s|'):format(k)
+          local tags_match = tags_text:match(tag_pat) --- @type string?
+          return tags_match and tags_match ~= 'deprecated.txt' and link_text or nil
+        end
+        local extra_desc = vim.iter(vim.spairs(events)):map(map_fn):join(',\n\t')
+        r.desc = r.desc:gsub('<PLACEHOLDER>', extra_desc)
```

Comment:
What flag decides whether the event is "outside of window context"? A brief comment here would be helpful.